src/ImageSharp/Formats/Tiff/TiffFrameMetadata.cs

```diff
@@ -0,0 +1,370 @@
+            {
+                var bits = this.ExifProfile.GetValue(ExifTag.BitsPerSample)?.Value;
+                if (bits == null)
+                {
+                    if (this.PhotometricInterpretation == TiffPhotometricInterpretation.WhiteIsZero
+                        || this.PhotometricInterpretation == TiffPhotometricInterpretation.BlackIsZero)
+                    {
+                        bits = new[] { (ushort)1 };
+                    }
+                    else
+                    {
+                        TiffThrowHelper.ThrowNotSupported("The TIFF BitsPerSample entry is missing.");
+                    }
+                }
+
+                return bits;
+            }
+        }
+
+        internal int BitsPerPixel
+        {
+            get
+            {
+                int bitsPerPixel = 0;
+                foreach (var bits in this.BitsPerSample)
+                {
+                    bitsPerPixel += bits;
+                }
+
+                return bitsPerPixel;
+            }
+        }
+
+        /// <summary>
+        /// Gets the compression scheme used on the image data.
+        /// </summary>
+        /// <value>The compression scheme used on the image data.</value>
+        public TiffCompression Compression
+        {
+            get
+            {
+                IExifValue<ushort> compression = this.ExifProfile.GetValue(ExifTag.Compression);
+                if (compression == null)
+                {
+                    return TiffCompression.None;
+                }
+
+                return (TiffCompression)compression.Value;
+            }
+        }
+
+        /// <summary>
+        /// Gets the color space of the image data.
+        /// </summary>
+        public TiffPhotometricInterpretation PhotometricInterpretation
+        {
+            get
+            {
+                IExifValue<ushort> photometricInterpretation = this.ExifProfile.GetValue(ExifTag.PhotometricInterpretation);
+                if (photometricInterpretation == null)
+                {
+                    return TiffPhotometricInterpretation.WhiteIsZero;
+                }
+
+                return (TiffPhotometricInterpretation)photometricInterpretation.Value;
+            }
+        }
+
+        /// <summary>
+        /// Gets the logical order of bits within a byte.
+        /// </summary>
+        internal TiffFillOrder FillOrder => (TiffFillOrder?)this.ExifProfile.GetValue(ExifTag.FillOrder)?.Value ?? TiffFillOrder.MostSignificantBitFirst;
+
+        /// <summary>
+        /// Gets or sets the a string that describes the subject of the image.
+        /// </summary>
+        public string ImageDescription
+        {
+            get => this.ExifProfile.GetValue(ExifTag.ImageDescription)?.Value;
+            set => this.ExifProfile.SetValue(ExifTag.ImageDescription, value);
+        }
+
+        /// <summary>
+        /// Gets or sets the scanner manufacturer.
+        /// </summary>
+        public string Make
```

Comment:
All these values shouldn't exist. They're not important enough to be distinct from querying the underlying ExifProfile.

Reply:
i have removed (most of them) with ffa38bc2